Task2.2/MatrixExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2._2
+{
+    public static class MatrixExtension
+    {
+        public static Matrix AddMatrix(this Matrix a, Matrix b)
+        {
+            var elements = new int[Math.Max(a.Size, b.Size)];
+            for (int i = 0; i < elements.Length; i++)
+            {
```

Comment:
Up to now it will work, but in real life it will be outofboudary exception for matrixes of different sizes. Please add 0 directly here if i > Size for one of matrixes

Reply:
But i have created such indexers, that if i will search for example an element [5,5] in any matrix and this matrix will have size for only 3 elements, than my indexer will return me just zero (according to the task requirements)